### PR TITLE
Improved the documentation of the TSN model

### PR DESCRIPTION
**Pull request by HydrogenSulfate**
1. add TSN's architecture image from it's paper
2. add some details.
3. add TSN's infer mode configuration.

### CODE REVIEW

File: docs/en/model_zoo/recognition/tsn.md [hunk 1]
```diff
@@ -2,69 +2,122 @@
...
+| :------: | :-------------: | :---------------: | :-----: | :---------: | :---: | :----------------------------------------------------------: |
+| ResNet50 |     Uniform     |       NCHW        |   25    |     224     | 69.81 | [TSN_k400.pdparams](https://videotag.bj.bcebos.com/PaddleVideo-release2.2/TSN_k400.pdparams) |
+
+## Inference
 
+### Export inference model
 
-当取如下参数时，在Kinetics400的validation数据集下评估精度如下:
+```bash
+python3.7 tools/export_model.py -c configs/recognition/tsn/tsn_k400_frames.yaml \
```
Comment (huangjun12) [inline docs/en/model_zoo/recognition/tsn.md:79]: 预测这块自己试过跑通了嘛？

File: docs/zh-CN/model_zoo/recognition/tsn.md [hunk 2]
```diff
@@ -1,107 +1,117 @@
...
 
-**训练策略：**
+当测试配置采用如下参数时，在Kinetics-400的validation数据集上的测试指标如下：
 
-*  采用[Momentum](https://www.paddlepaddle.org.cn/documentation/docs/zh/develop/api/paddle/optimizer/momentum/Momentum_cn.html#momentum)优化算法训练，momentum值设定为0.9。
-*  [l2_decay](https://www.paddlepaddle.org.cn/documentation/docs/zh/develop/api/paddle/regularizer/L2Decay_cn.html#l2decay)权重衰减系数为1e-4。
-*  学习率在训练的总epoch数的15（1/3）和30（2/3）时分别做0.1倍的衰减。
+| backbone | Sampling method | Training Strategy | num_seg | target_size | Top-1 | checkpoints                                                  |
+| :------: | :-------------: | :---------------: | :-----: | :---------: | :---: | ------------------------------------------------------------ |
+| ResNet50 |     Uniform     |       NCHW        |   25    |     224     | 69.81 | [TSN_k400.pdparams](https://videotag.bj.bcebos.com/PaddleVideo-release2.2/TSN_k400.pdparams) |
```
Comment (huangjun12) [inline docs/zh-CN/model_zoo/recognition/tsn.md]: Uniform改成10 crops

File: docs/en/model_zoo/recognition/tsn.md (hunk 1 above)
Comment (HydrogenSulfate) [reply docs/en/model_zoo/recognition/tsn.md:79]: 已修改

**Comment by huangjun12** [comment]
LGTM

**Pull request merged**